docs/cudf/source/conf.py

```diff
@@ -195,11 +279,13 @@
 
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {
-    "python": ("https://docs.python.org/3", None),
     "cupy": ("https://docs.cupy.dev/en/stable/", None),
+    "dlpack": ("https://dmlc.github.io/dlpack/latest/", None),
     "numpy": ("https://numpy.org/doc/stable", None),
-    "pyarrow": ("https://arrow.apache.org/docs/", None),
     "pandas": ("https://pandas.pydata.org/docs/", None),
+    "pyarrow": ("https://arrow.apache.org/docs/", None),
+    "python": ("https://docs.python.org/3", None),
+    "rmm": ("https://docs.rapids.ai/api/rmm/nightly/", None),
```

Comment:
Does this line mean we can remove the Doxygen step that fetches the `rmm.tag` file below?

- https://github.com/rapidsai/cudf/blob/8dca25c782bbe239ed6e9b6317cc3a01b15a2b42/ci/build_docs.sh#L35

Reply:
Yes, we don't need that anymore.